src/coreclr/jit/lowerarmarch.cpp

```diff
@@ -292,6 +292,30 @@ GenTree* Lowering::LowerMul(GenTreeOp* mul)
 //
 GenTree* Lowering::LowerBinaryArithmetic(GenTreeOp* binOp)
 {
+    if (comp->opts.OptimizationEnabled() && binOp->OperIs(GT_AND))
```

Comment:
Could you make sure there were no diffs on ARM (32 bit)? There should not be any, but SPMI in CI implies there could have been.

Reply:
How do I do that?

Reply:
Build the Jits targeting x86, there will be an AltJit there: `clrjit_universal_arm_x86.dll`, you can use it with SPMI to obtain ARM diffs (`superpmi.py asmdiffs -target_os Linux -target_arch arm -arch x86 -jit_name clrjit_universal_arm_x86.dll`).

Reply:
Ran that commandline, no asm diffs.